src/Umbraco.Web.UI.Client/src/less/forms.less

```diff
@@ -297,11 +297,11 @@ select[size] {
 }
 
 // Focus for select, file, radio, and checkbox
-select:focus,
-input[type="file"]:focus,
-input[type="radio"]:focus,
-input[type="checkbox"]:focus {
-  .tab-focus();
+select,
+input[type="file"],
+input[type="radio"],
+input[type="checkbox"] {
+  .umb-outline();
```

Comment:
@nielslyngsoe this use `umb-outline` class mixin, but this has by default `outline: 0`. At the moment `select` doesn't have outline as previous.

![image](https://user-images.githubusercontent.com/2919859/73868294-7a01b980-4848-11ea-933a-427d546a8754.png)

Reply:
@bjarnef your right, this does not make sense. I'm removing "select" from this css-selector. PR: https://github.com/umbraco/Umbraco-CMS/pull/7788